lib/enigma.rb

```diff
@@ -1,9 +1,22 @@
 require 'date'
+require_relative 'shifts'
 class Enigma
-  attr_reader :alphabet
+  attr_reader :alphabet, :key, :date
 
   def initialize
     @alphabet = Array("a".."z")<< " "
+    @key = rand(99999).to_s
+    @date = Date.today.strftime("%d%m%y")
+  end
```

Comment:
Moved the default values for both `key` and `date` from `Shifts` to `Enigma` for better integration.